graphql-ruby, puma, rack

tak1n opened the pull request:
Feel free to leverage or close as you wish :)

jgillich commented:
Ow I'm sorry, I also added graphql-ruby a few minutes ago https://github.com/graphql-crystal/benchmarks/tree/main/graphql-ruby

But I wasn't able to figure out how to disable request logging, and we probably have to execute it with `puma -w <cpu>` like you did? I'm not experienced with Ruby at all, would be great if you could help!

Also I think it makes more sense to use common web server frameworks and not raw HTTP servers, that's why I'm using Sinatra

tak1n commented:
> Ow I'm sorry, I also added graphql-ruby a few minutes ago https://github.com/graphql-crystal/benchmarks/tree/main/graphql-ruby

No worries, that's why I renamed it to `graphql-ruby-rack-puma`. Valid point in comparing frameworks instead of low level HTTP servers.

> I'm not experienced with Ruby at all, would be great if you could help!

Will open a new PR to adapt the sinatra based application. Are you interested in another example comparing with [roda](http://roda.jeremyevans.net/index.html)?

jgillich commented:
If roda is a popular alternative to sinatra that is also faster then we can switch to it entirely. Not super interested in having multiple web servers per lib because there would be near unlimited combinations
